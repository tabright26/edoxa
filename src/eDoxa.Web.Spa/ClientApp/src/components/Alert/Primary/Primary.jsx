import React from "react";
import { Alert } from "reactstrap";

const SuccessAlert = ({ heading, body, footer = null }) => (
  <Alert color="primary">
    <h4 className="alert-heading">{heading}</h4>
    <p className={footer ? null : "mb-0"}>{body}</p>
    {footer ? (
      <>
        <hr />
        <p className="mb-0">{footer}</p>
      </>
    ) : null}
  </Alert>
);

export default SuccessAlert;
