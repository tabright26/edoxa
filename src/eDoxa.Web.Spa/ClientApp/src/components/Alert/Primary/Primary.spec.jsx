import React from "react";
import renderer from "react-test-renderer";
import Primary from "./Primary";

it("renders without crashing", () => {
  const tree = renderer.create(<Primary />).toJSON();
  expect(tree).toMatchSnapshot();
});
