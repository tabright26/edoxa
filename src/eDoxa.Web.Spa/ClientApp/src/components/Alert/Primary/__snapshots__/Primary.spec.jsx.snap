// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="alert alert-primary fade"
  role="alert"
>
  <h4
    className="alert-heading"
  />
  <p
    className="mb-0"
  />
</div>
`;
